config-files/UM/um_env_to_yaml.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# =========================================================================================
+# Convert depreciated um_env.py configuration files for ESM1.5 simulations into yaml, for
+# use with the newer versions of payu.
+#
+# To run:
+#   python um_env_to_yaml.py <input_filepath> [--ofile <output_filepath>]
+#
+# If --ofile is not specified, it defaults to the same directory as  <input_filepath>
+# with filename "um_env.yaml"
+#
+# Contact:
+#   Spencer Wong <spencer.wong@anu.edu.au>
+#
+# Dependencies:
+#    pyyaml
```

Comment:
It's been a while...is this a standard library? If it isn't should there be a `requirements.txt` that people can `pip install` these dependencies?

Reply:
Good idea! I don't think it is a standard library – I've added in a `requirements.txt` file.